extensions-contrib/distinctcount/src/main/java/io/druid/query/aggregation/distinctcount/DistinctCountAggregator.java

```diff
@@ -19,16 +19,19 @@
 
 package io.druid.query.aggregation.distinctcount;
 
+import com.google.common.base.Preconditions;
 import io.druid.collections.bitmap.MutableBitmap;
+import io.druid.common.config.NullHandling;
 import io.druid.query.aggregation.Aggregator;
 import io.druid.segment.DimensionSelector;
 import io.druid.segment.data.IndexedInts;
 
 public class DistinctCountAggregator implements Aggregator
```

Comment:
Please add a javadoc to `DistinctCountAggregator` stating that "if performance of this class appears to be a bottleneck for somebody, one simple way to improve it is to split it into two different classes, one that is used when {@link NullHandling.useDefaultValuesForNull()} is false, and one - when it's true, moving this computation out of the tight loop".

Reply:
done.